codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/generators/http/ServerResponseBindingGenerator.kt

```diff
@@ -94,7 +95,7 @@ class ServerResponseBeforeRenderingHeadersHttpBindingCustomization(val codegenCo
                     codegenContext.settings.codegenConfig.publicConstrainedTypes,
                 )
             ) {
-                if (section.context.shape is IntegerShape || section.context.shape is ShortShape || section.context.shape is LongShape || section.context.shape is ByteShape) {
+                if (section.context.shape is IntegerShape || section.context.shape is ShortShape || section.context.shape is LongShape || section.context.shape is ByteShape || section.context.shape is CollectionShape) {
```

Comment:
This line is very long. Can we break it in multiple shorter lines?

We can also merge this conditional with the `workingWithPublicConstrainedWrapperTupleType` one.

Reply:
Done in https://github.com/awslabs/smithy-rs/pull/2028/commits/01cfe2386e855f4dade0e9ae17cb1a156f494997.